api4/user_test.go

```diff
@@ -5678,7 +5678,8 @@ func TestMaintainUnreadMentionsInThread(t *testing.T) {
 		*cfg.ServiceSettings.ThreadAutoFollow = true
 		*cfg.ServiceSettings.CollapsedThreads = model.COLLAPSED_THREADS_DEFAULT_ON
 	})
-	checkThreadList := func(client *model.Client4, userId string, expectedMentions, expectedThreads int) (*model.Threads, *model.Response) {
+
+	checkThreadList := func(client *model.Client4, userId string, expectedThreads int) (*model.Threads, *model.Response) {
```

Comment:
Clearly we are not checking the expectedMentions, but... should we? @reflog

Reply:
@jespino yeah... seems like a buggy test. I'll open a ticket for myself to fix this.